app/_locales/en/messages.json

```diff
@@ -347,6 +350,12 @@
   "backupNow": {
     "message": "Backup now"
   },
+  "backupUserData": {
+    "message": "Backup your data"
+  },
+  "backupUserDataDescription": {
+    "message": "Your data contains preferences and account addresses"
```

Comment:
I think something like this might be more transparent for the user? 
`You can backup user settings containing preferences and account addresses into a JSON file.`